deployment/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+git clone https://wroclawjug:${github_token}@github.com/wroclawjug/wroclawjug.github.io.git
+cd wroclawjug.github.io
+rm -r losowarka/*
+cp ../dist/* losowarka/
+git add -A
+git commit || true
```

Comment:
I'm afraid it will open an editor to type commit message and cause an error. Could you add `-m` and some commit message please?

Reply:
Fixed.